test/acutest.h

```diff
@@ -1830,7 +1830,7 @@ main(int argc, char** argv)
 
     acutest_test_data_ = (struct acutest_test_data_*)calloc(acutest_list_size_, sizeof(struct acutest_test_data_));
     if(acutest_test_data_ == NULL) {
-        fprintf(stderr, "Out of memory.\n");
+        fprintf(stderr, "%s\n", strerror(errno));
```

Comment:
This is 3rd-party source -- https://github.com/mity/acutest
Please revert.

Reply:
Done.